runtime.md

```diff
@@ -13,6 +13,13 @@ The state of a container MUST include, at least, the following properties:
 * **`id`**: (string) is the container's ID.
 This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
+* **`status`**: (string) is the runtime state of the container.
+The value MAY be one of:
+  * `created` : the container has been created but the user-specified process has not yet been executed
```

Comment:
I changed all "user-specific process" to "user-specific code" so the doc is now consistent (with itself anyway).